docs/src/pages/es/getting-started.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Empezando
```

Comment:
```suggestion
title: Empezando
lang: "es"
```

Could you add language codes to the entries so that the document gets `<html lang="es">`?

Reply:
Yes, I apologize. I did see that on discord and  forgot to add that in.